stable/grafana/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: grafana
-version: 5.3.2
+version: 5.3.3
```

Comment:
Please bump to the latest version. Thank you.

Reply:
Rebased & bumped.